pilates/utils/io.py

```diff
@@ -17,24 +17,22 @@ def read_datastore(settings, year=None, warm_start=False):
 
     region = settings['region']
     region_id = settings['region_to_region_id'][region]
-    usim_local_data_folder = settings['usim_local_data_folder']
+    data_dir = settings['data_folder']
```

Comment:
same comment with `data_dir`

Reply:
... not sure which other comment you were refering to - but this change is to make sure that the usim data folder is specified fully (assuming it is relative to data dir and then building a full [absolute] path for it).

Reply:
no need to change anything.